packages/table/preview/index.tsx

```diff
@@ -42,6 +42,7 @@ ReactDOM.render(<Nav selected="perf" />, document.getElementById("nav"));
 
 import { IFocusedCellCoordinates } from "../src/common/cell";
 import { IColumnIndices, IRowIndices } from "../src/common/grid";
+import { RenderOptimizationMode } from "../src/common/renderOptimizationMode";
```

Comment:
Can we just call it render mode? And mention that we do optimizations in the docs? That way we can get rid of the extra mouthful.

Reply:
Sure